sympy/core/evalf.py

```diff
@@ -944,7 +997,7 @@ def evalf_log(expr, prec, options):
         return re, None, re_acc, None
 
 
-def evalf_atan(v, prec, options):
+def evalf_atan(v: 'atan', prec: int, options: tDict[str, Any]) -> TEMP_RESULT:
```

Comment:
Maybe a type should be defined at the top for options so it doesn't get repeated everywhere. I think we can be more precise than Any here e.g.:
```
tDict[str, Optional[tUnion[bool, int, Dict[Basic, Sympifiable]]]
````
(I realise we don't yet have sympifiable)

Reply:
This produces many errors as different keys are associated with different types. TypedDict is needed, which is new in python 3.8